CHANGELOG.md

```diff
@@ -3,6 +3,30 @@
 Documentation for hipSPARSE is available at
 [https://rocm.docs.amd.com/projects/hipSPARSE/en/latest/](https://rocm.docs.amd.com/projects/hipSPARSE/en/latest/).
 
+## hipSPARSE 3.0.1 for ROCm 6.2.0
+
```

Comment:
We need to bump the version. Can you also do this in this PR?

Reply:
Fixed